fiftyone/core/odm/utils.py

```
@@ -713,20 +714,20 @@ def load_dataset(id=None, name=None):
     if name:
-        return fod.load_dataset(name)
+        return fod.load_dataset(name, reload=reload)
 
     if not id:
         raise ValueError("Must provide either id or name")
 
     db = foo.get_db_conn()
     try:
-        uid = ObjectId(id)
+        _id = ObjectId(id)
     except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)